cardano-crypto-praos/cbits/vrf03/vrf.c

```diff
@@ -1,37 +1,16 @@
-/*
-Copyright (c) 2018 Algorand LLC
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
-*/
-
 #include "crypto_vrf_ietfdraft03.h"
+#include "../crypto_vrf.h"
 
 size_t
-crypto_vrf_ietfdraft03_publickeybytes(void)
+crypto_vrf_ietfdraft03_proofbytes(void)
 {
-    return crypto_vrf_ietfdraft03_PUBLICKEYBYTES;
+    return crypto_vrf_ietfdraft03_BYTES;
```

Comment:
Maybe in the cbits of cardano-base we don't really need to worry about backwards compatibility, right? As the only code using these functions is cardano-base itself. So maybe we could fully remove `crypto_vrf_ietfdraft03_proofbytes`. The only place were we need to keep it for backwards compatibility is in the libsodium library. WDYT?

Reply:
Yeah, I agree, in that case it should be fine to break this backwards compatibility. 

However, we need to not to forget to update this function in our fork of libsodium too

Reply:
Mmmh, you are right about the function in libsodium's fork. I don't its worth doing an update on libsodium and nix just for this naming issue. I'll fall back to your initial suggestion.

Reply:
Well, partly, because we cannot change the Haskell FFI bindings, as otherwise we would need to change the fork of libsodium as well.

Reply:
Well, since you already changed the `crypto_vrf_ietfdraft13_bytes_batchcompat` from `crypto_vrf_batchcompat_ietfdraft13_bytes`, I think we already have to change the libsodium fork and make a nix release

With respect to the latter change, I think it is best to get this naming done right once before the official release, this way we'll have less chance of dealing with deprecation and renaming in the future.

Reply:
Yes, good point, I'm going to have to do a nix bump anyways. I'll change it.